Gruntfile.js

```diff
@@ -8,7 +8,7 @@ module.exports = function (grunt) {
             "node_modules"
           ],
           modifyVars: {
-            libsPath: ""
+            libsPath: "''"
```

Comment:
@natocTo vedel by si pls objasnit, co robi tato zmena?

Reply:
Co jsem pochopil tak když chci použít jako default prázdný string musím to zapsat takto. 

Bez toho se to vyhodnotí asi jinak a pak to nejde použít jako var do stringu a končí to chybou. Můžeš zkusit to dát pryč a udělat build.